### PR TITLE
Add workflow and test ckan 2.10

### PR DESCRIPTION
**Pull request by avdata99**
Related to [CKAN#6787](https://github.com/ckan/ckan/issues/6787)

### CODE REVIEW

File: dev-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+flask==1.0.2
```
Comment (amercader) [inline dev-requirements.txt:1]: Why do we need Flask as a dev requirement?

File: requirements.txt [hunk 2]
```diff
@@ -1,2 +1,3 @@
 six>=1.11.0
-ckantoolkit>=0.0.2
\ No newline at end of file
+ckantoolkit>=0.0.2
+itsdangerous==2.0.1
```
Comment (amercader) [inline requirements.txt:3]: AFAICT this is Flask related requirement, shouldn't we be using whatever version CKAN core installs?

File: test-nose.ini [hunk 3]
```diff
@@ -0,0 +1,60 @@
+[DEFAULT]
```
Comment (amercader) [inline test-nose.ini:1]: This file is not actually needed right?

File: dev-requirements.txt [hunk 4]
```diff
@@ -0,0 +1,4 @@
...
+mock
+pytest-ckan
+pytest-cov
```
Comment (amercader) [inline dev-requirements.txt:4]: I guess we need ckanext-scheming as a requirement here

File: test-nose.ini (hunk 3 above)
Comment (avdata99) [reply test-nose.ini]: Deleted

File: dev-requirements.txt (hunk 4 above)
Comment (avdata99) [reply dev-requirements.txt:3]: Added to the requirements file

File: requirements.txt [hunk 5]
```diff
@@ -1,2 +1,3 @@
-six>=1.11.0
-ckantoolkit>=0.0.2
\ No newline at end of file
+-e git+https://github.com/ckan/ckanext-scheming.git#egg=ckanext-scheming
+six==1.16.0
+ckantoolkit==0.0.7
```
Comment (amercader) [inline requirements.txt:3]:
```suggestion
six
ckantoolkit>=0.0.7
```

File: dev-requirements.txt (hunk 1 above)
Comment (avdata99) [reply dev-requirements.txt]: removed

File: requirements.txt (hunk 2 above)
Comment (avdata99) [reply requirements.txt]: updated

**Pull request merged**